[DOCS] Adds Google Vertex AI tutorial (#112339)
elastic/elasticsearch: docs/reference/tab-widgets/inference-api/infer-api-reindex.asciidoc

// end::azure-ai-studio[]

> // tag::google-vertex-ai[]

Reviewer: Not part of this PR, but should we link to the reindex API in general in this step and add maybe one sentence why we're using it (basically simulating ingestion)?

[source,console]
----
POST _reindex?wait_for_completion=false
{
  "source": {
    "index": "test-data",
    "size": 50 <1>
  },
  "dest": {
    "index": "google-vertex-ai-embeddings",
    "pipeline": "google_vertex_ai_embeddings"
  }
}
----
// TEST[skip:TBD]

> <1> The default batch size for reindexing is 1000. Reducing `size` to a smaller
> number makes the update of the reindexing process quicker which enables you to
> follow the progress closely and detect errors early.

Reviewer 2: ```suggestion
<1> The default batch size for reindexing is 1000. Reducing `size` will make updates to the reindexing process faster. This enables you to
follow the progress closely and detect errors early.
```
